instrumentation/nocode/src/test/java/com/splunk/opentelemetry/instrumentation/nocode/JexlTest.java

```diff
@@ -41,10 +41,13 @@ class JSPSTest {
     param0.add("present");
   }
 
-  static Stream<Arguments> jspsToExpected() {
+  private static String evalJexl(String jexl, Object thiz, Object[] params) {
+    return new JexlEvaluator().evaluate(jexl, thiz, params);
+  }
+
+  static Stream<Arguments> jexlToExpected() {
     return Stream.of(
-        // Might be nice to support a bare "param0" or "this" but as a workaround can always use
-        // "this.toString()"
+        arguments("this", "{key=value}"),
```

Comment:
an alternative to using expressions would be to use templates (I think this is what they are called in jexl) where you have something like `Hello ${this.name}`

Reply:
Yeah, jexl has a lot of features; I tried to keep things more java-y than expression-y.  When this gets more mature/field-tested I'll add docs for common scenarios, the names of the provided variables, etc.

Reply:
Unfortunately the behavior is hard to change once this feature is made public so we'll have to consider that we are probably going to be stuck with whatever we come up here for a long time.